test/lit.cfg

```diff
@@ -2107,6 +2107,13 @@ config.substitutions.append(('%string_processing_module', string_processing_modu
 config.substitutions.append(('%/string_processing_module',
                              '/'.join(os.path.normpath(string_processing_module).split(os.sep))))
 
+# Add 'pointer_bounds_module' as the path to the _PointerBounds .swiftmodule file
+pointer_bounds_module = os.path.join(stdlib_dir, "_PointerBounds.swiftmodule",
```

Comment:
I'm surprised you needed this. If we replace the `_PointerBounds` module with an `#if` in the `Swift` module, you won't need it.